docs/_api/plugins.md

```diff
@@ -745,54 +753,55 @@ Returns **[Function](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Ref
 
 ### cpuUsageThrottle
 
-cpuUsageThrottle is a middleware that rejects a variable number of requests
-(between 0% and 100%) based on a historical view of CPU utilization of a
-Node.js process. Essentially, this plugin allows you to define what
-constitutes a saturated Node.js process via CPU utilization and it will
-handle dropping a % of requests based on that definiton. This is useful when
-you would like to keep CPU bound tasks from piling up causing an increased
+cpuUsageThrottle is a middleware that rejects a variable number of requests 
```

Comment:
Trailing spaces? I think your editor may not bet stripping these?

Reply:
`docs/_api/*.md` is auto-generated, I fixed whitespaces at other places.